admin_manual/release_notes.rst

```diff
@@ -0,0 +1,266 @@
+================================
+ownCloud |version| Release Notes
+================================
+
+Recommended Setup for Running ownCloud
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ownCloud runs on multiple operating systems and HTTP servers. For best 
+performance, stability, support, and full functionality we recommend:
+
+* Red Hat Enterprise Linux 7
+* MySQL/MariaDB
+* PHP 5.4 +
+* Apache 2.4
+
+Supported Platforms
+^^^^^^^^^^^^^^^^^^^
+
+* Hypervisors: Hyper-V, VMware ESX, Xen, KVM
+* Server: Windows 2008R2; 2012, Linux (Debian 6, RHEL / Centos 6 and 7, Ubuntu 
```

Comment:
Please say 12.04 LTS, 14.04 LTS and 14.10 ... I never read of Ubuntu 12 before :P